shared/rakefiles/xk_util.rake

```diff
@@ -258,13 +271,16 @@ task :restore_snapshot_from_image_file, [:snapshot_files] => [@gcp_creds_file, :
   end
 
   snapshot_files.each do |snapshot_file|
+    # The snapshot_name must be something like:
+    # database-storage-couchdb-couchdb-0-060619-195849 from the file name:
+    # 2019-06-27_154922-pv-database-storage-couchdb-couchdb-0-060619-195849.tar.gz
     snapshot_name = snapshot_file[/database-storage-couchdb-couchdb-\d-\d+-\d+/, 0]
     sh "#{@exekube_cmd} sh -c ' \
-    gcloud compute images create image-disk-pv-#{snapshot_name} --source-uri=#{snapshot_file}
-    gcloud compute disks create disk-pv-#{snapshot_name} --zone=#{pv_zones[snapshot_name[/(([A-Za-z]+-)+[\d])/,0]]} --image=image-disk-pv-#{snapshot_name}
-    gcloud compute disks snapshot disk-pv-#{snapshot_name} --zone=#{pv_zones[snapshot_name[/(([A-Za-z]+-)+[\d])/,0]]} --snapshot-names external-pv-#{snapshot_name}
-    gcloud -q compute images delete image-disk-pv-#{snapshot_name}
-    gcloud -q compute disks delete disk-pv-#{snapshot_name} --zone=#{pv_zones[snapshot_name[/(([A-Za-z]+-)+[\d])/,0]]}
+      gcloud compute images create image-disk-pv-#{snapshot_name} --source-uri=#{snapshot_file}
+      gcloud compute disks create disk-pv-#{snapshot_name} --zone=#{pv_zones[snapshot_name[/(([A-Za-z]+-)+[\d])/,0]]} --image=image-disk-pv-#{snapshot_name}
```

Comment:
Maybe setting the zone name to a variable once would avoid multiple `pv_zones[snapshot_name[/(([A-Za-z]+-)+[\d])/,0]]` calls and make it a bit more obvious what is that expression for.